app/web/features/auth/volunteer/VolunteerManagement.tsx

```diff
@@ -0,0 +1,509 @@
+import {
+  Avatar,
+  Card,
+  CardContent,
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  InputLabel,
+  MenuItem,
+  Select,
+  styled,
+  Typography,
+} from "@mui/material";
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import CenteredSpinner from "components/CenteredSpinner/CenteredSpinner";
+import { EmailIcon, GlobeIcon, LinkedInIcon, PinIcon } from "components/Icons";
+import IconText from "components/IconText";
+import StyledLink from "components/StyledLink";
+import TextField from "components/TextField";
+import { volunteerInfoQueryKey } from "features/queryKeys";
+import useCurrentUser from "features/userQueries/useCurrentUser";
+import { RpcError } from "grpc-web";
+import { Trans, useTranslation } from "i18n";
+import { AUTH } from "i18n/namespaces";
+import { GetAccountInfoRes, GetMyVolunteerInfoRes } from "proto/account_pb";
+import { Controller, useForm } from "react-hook-form";
+import { volunteerNotAVolunteerFormUrl } from "routes";
+import { service } from "service";
+import { theme } from "theme";
+
+const StyledForm = styled("form")(() => ({
+  marginTop: theme.spacing(2),
+  "& > * + *": {
+    marginBlockStart: theme.spacing(2),
+  },
+}));
+
+const VolunteerCardWrapper = styled("div")(() => ({
+  marginTop: theme.spacing(2),
+  marginBottom: theme.spacing(2),
+  maxWidth: "400px",
+}));
+
+const VolunteerCard = styled(Card)(() => ({
+  height: "13rem",
+  border: `1px solid ${theme.palette.grey[400]}`,
+}));
+
+const VolunteerCardContent = styled(CardContent)(() => ({
+  display: "flex",
+}));
+
+const DetailDiv = styled("div")(() => ({
+  display: "flex",
+  flexFlow: "column nowrap",
+  gap: theme.spacing(0.5),
+  padding: theme.spacing(1, 2),
+}));
+
+const StyledAvatar = styled(Avatar)(() => ({
+  width: theme.typography.pxToRem(96),
+  height: theme.typography.pxToRem(96),
+}));
+
+const VolunteerStatus = styled("div")(() => ({
+  marginTop: theme.spacing(1),
+  display: "flex",
+  flexDirection: "column",
+  gap: theme.spacing(0.5),
+}));
+
+const FormRow = styled("div")(() => ({
+  display: "flex",
+  gap: theme.spacing(2),
+  flexWrap: "wrap",
+  alignItems: "flex-start",
+}));
+
+type VolunteerManagementProps = {
+  accountInfo: GetAccountInfoRes.AsObject;
+  className?: string;
+};
+
+interface VolunteerFormData {
+  overrideName: boolean;
+  displayName: string;
+  overrideLocation: boolean;
+  displayLocation: string;
+  showOnTeamPage: boolean;
+  linkType: string;
```

Comment:
```suggestion
  linkType: typeof LINK_TYPES[number];
```

I think this will fix it so you don't need the `as const`.